website/docs/r/network_interface.html.markdown

```diff
@@ -91,11 +92,13 @@ The `ip_configuration` block supports:
 
 * `load_balancer_inbound_nat_rules_ids` - (Optional) List of Load Balancer Inbound Nat Rules IDs involving this NIC
 
+* `primary` - (Optional) Is this the Primary Network Interface? If set to `true` this should be the first `ip_configuration` in the array.
```

Comment:
so, the user can submit the second (or later) ip_configuration as `primary` and that'll update but lead to a perpetual diff, as it's returned first in the response; we could _probably_ do something there, but this mirrors the UI

The alternative I considered here was just assuming the first would be the primary, but that may not necessarily be optimal? WDYT?